requirements.txt

```diff
@@ -1,2 +1,3 @@
-requests==2.28.1
+httpx==0.23.0
```

Comment:
Will also need an `extra_requires` entry in setup.py, so people ensure they have it with `pip install python-gitlab[httpx]`.

Reply:
Added to the "install_requires" key where the reqests package is listed

Reply:
Ah, see my other comment about optional dependencies - IMO extras are a good place for this to avoid adding it by default :)

Reply:
Yeah. Pushed a fix.